app/containers/TrackInfo/index.js

```diff
@@ -37,22 +87,42 @@ export function TrackInfo({ dispatchGetSongDetails, dispatchClearSongDetails, so
         <title>TrackInfo</title>
         <meta name="description" content="Description of TrackInfo" />
       </Helmet>
-      <pre>{JSON.stringify(songInfo, null, 2)}</pre>
+      <TrackInfoContainer>
+        <Artwork src={songInfo.artworkUrl250} width="250px" height="250px" alt={songInfo.trackName} />
+        <TrackTitle>{songInfo.trackName}</TrackTitle>
+        <TrackArtist href={songInfo.artistViewUrl} target="__blank" rel="noreferrer noopener">
+          {songInfo.artistName}
+        </TrackArtist>
+        <GenreName>
+          {songInfo.primaryGenreName} · {songInfo.releaseYear}
+        </GenreName>
+        <ButtonsContainer>
+          <Button
+            type="primary"
+            shape="round"
+            href={songInfo.trackViewUrl}
+            target="__blank"
+            rel="noreferrer noopener"
+            size="medium"
+            icon={<CaretRightOutlined />}
+          >
+            Play
```

Comment:
Please add translation for this button text.. Usually we create a component that takes in the translation id to render the button text.